src/Symfony/Component/Uid/Ulid.php

```diff
@@ -124,10 +124,21 @@ public function getTime(): float
         return BinaryUtil::toBase(hex2bin($time), BinaryUtil::BASE10) / 1000;
     }
 
-    private static function generate(): string
+    /**
+     * @param float|null $timestamp Seconds since the Unix epoch 1970-01-01 00:00:00
```

Comment:
Should we indicate that the timestamp will be rounded to milliseconds precision (same in the factory)?

Reply:
no need